Makefile

```diff
@@ -67,7 +67,9 @@ coverage: ## check code coverage quickly with the default Python
 docs: ## generate Sphinx HTML documentation, including API docs
 	# Warning: as the sphinx-apidoc is NOT being run on the RTD server the workaround we have
 	# is to commit the (currently 8) rst files it generates.
-	sphinx-apidoc -d 10 -o docs/ ravenpy
+	rm -f docs/ravenpy*.rst
```

Comment:
That looks dangerous, no ?

Reply:
We use the exact line in all other docs-making entries (e.g. https://github.com/Ouranosinc/xclim/blob/b0ed02f3d8bc25667b0f7d87cfe294177ce76269/Makefile#L87). I could remove the `-f` for safety (this only affects files made when `sphinx-apidoc` is run, typically by the user who last called it). What do you think?

Reply:
Actually, I'd keep the "-f" as some shells (`zsh`/`fish`) can potentially prompt for user input when removing multiple files. There's no recursion happening here. Should I instead make a mention in the docs to not name any doc files after the slug name (`ravenpy`)?

Reply:
Sure, that seems reasonable.